### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by Saikatssd**
Fixed some typos. Please Review.

    What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)
    Docs Update
    Why was this change needed? (You can also link to an open issue here)
    To improve the quality of documentation.
    Other information:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 ## What is Backstage?
 
-[Backstage](https://backstage.io/) is an open platform for building developer portals. Powered by a centralized software catalog, Backstage restores order to your microservices and infrastructure and enables your product teams to ship high-quality code quickly — without compromising autonomy.
+[Backstage](https://backstage.io/) is an open platform for building developer portals. Powered by a centralized software catalogue, Backstage restores order to your microservices and infrastructure and enables your product teams to ship high-quality code quickly — without compromising autonomy.
```
Comment (awanlin) [inline README.md:15]: This probably shouldn't be changed. Catalog is the term used in all the docs and code.

**Review by awanlin — CHANGES_REQUESTED** [review]
Hi @Saikatssd, thanks for the contribution. Just left a comment about one of the changes

**Review by Rugvip — APPROVED** [review]
Thanks! 🎉

**Comment by suuus** [comment]
Thank you for contributing during Hacktoberfest 🍂! You can claim the Backstage Hacktoberfest Holopin at https://bck.st/hacktoberfest-holopin 🙌🏻

**Pull request merged**